scheduler/system_sched.go

```diff
@@ -83,6 +83,12 @@ func (s *SystemScheduler) Process(eval *structs.Evaluation) error {
 		s.queuedAllocs)
 }
 
+// isStoppedJob returns if the scheduling is for a stopped job and the scheduler
+// should stop all its allocations.
+func (s *SystemScheduler) isStoppedJob() bool {
```

Comment:
You could define this on the `Job` to avoid double defining in both schedulers (e.g. just check for nil receiver pointer)